hack/new-release.sh

```diff
@@ -0,0 +1,182 @@
+#!/usr/bin/env bash
+# Copyright (c) 2017-present SIGHUP s.r.l All rights reserved.
+# Use of this source code is governed by a BSD-style
+# license that can be found in the LICENSE file.
+
+
+set -e
+set -u
+set -o pipefail
+
+UPSTREAM_REPO="https://github.com/prometheus-operator/kube-prometheus"
+
+if [ $# -ne 2 ]; then
+  echo "Usage: $(basename $0) <upstream_release> <fury_package>" 1>&2
+  echo "Where:" 1>&2
+  echo "  - <upstream_release> must be a valid reference in the ${UPSTREAM_REPO} Git repository" 1>&2
+  echo "  - <fury_package> must be one of:" 1>&2
+  echo "    - aks-sm" 1>&2
+  echo "    - alertmanager-operated" 1>&2
+  echo "    - blackbox-exporter" 1>&2
+  echo "    - eks-sm" 1>&2
+  echo "    - gke-sm" 1>&2
+  echo "    - grafana" 1>&2
+  echo "    - kube-state-metrics" 1>&2
+  echo "    - kubeadm-sm" 1>&2
+  echo "    - node-exporter" 1>&2
+  echo "    - prometheus-adapter" 1>&2
+  echo "    - prometheus-operated" 1>&2
+  echo "    - prometheus-operator" 1>&2
+  exit 1
+fi
+
+UPSTREAM_RELEASE="${1}"
+FURY_MODULE="${2}"
+
+WORK_DIR="$(mktemp -d kube.prometheus.XXXXXX -p /tmp)"
+KATALOG_PATH="$(dirname $(readlink -e "${0}"))/../katalog"
```

Comment:
shellcheck is complaining about these missing quotes:
```suggestion
KATALOG_PATH="$(dirname "$(readlink -e "${0}")")/../katalog"
```

(there are other similar warnings from shellcheck but the previous is the only blocking one)

nit: I'd suggest renaming the folder from `hack` to `utils` or something more suitable and the name of the script to `pull-upstream.sh` or similar, from what I understand the script does not prepare a new release of the module/package but updates all the files with the upstream versions.